optuna/integration/skorch.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any
+
+import optuna
+
+with optuna._imports.try_import() as _imports:
+    from skorch.callbacks import Callback
+    from skorch.net import NeuralNet
+
+if not _imports.is_successful():
+    Callback = object  # NOQA
+
+
+class SkorchPruningCallback(Callback):
```

Comment:
TODO: Add `versionadded` 

ref: https://github.com/optuna/optuna/wiki/Coding-Style-Conventions#write-versionadded-directive-to-the-doc-of-new-features